core/src/main/java/com/scalar/db/api/Put.java

```diff
@@ -30,7 +31,7 @@
  */
 @NotThreadSafe
 public class Put extends Mutation {
-  private final Map<String, Optional<Value<?>>> values;
+  private final Map<String, Column<?>> columns;
```

Comment:
I changed `Map<String, Optional<Value<?>>> values` to `Map<String, Column<?>> columns` here.